app/src/main/java/com/example/weather/ui/adapter/DailyAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.weather.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.weather.data.model.entity.WeatherBasic
+import com.example.weather.databinding.LayoutItemDailyBinding
+import com.example.weather.utils.Utils.getIcon
+import com.example.weather.utils.ext.kelvinToCelsius
+import com.example.weather.utils.ext.unixTimestampToDateMonthString
+import com.example.weather.utils.ext.unixTimestampToHourString
+
+class DailyAdapter : RecyclerView.Adapter<DailyAdapter.ViewHolder>() {
+
+    private val mListWeathers by lazy { mutableListOf<WeatherBasic>() }
```

Comment:
```suggestion
    private val mListWeathers by lazy { mutableListOf<WeatherBasic>() }
```
mListWeathers -> typo

sửa thành ``mWeathers`` hoặc ``mListWeather``